src/components/formRow/formRow.comp.tsx

```diff
@@ -172,19 +174,34 @@ export const FormRow = withAppContext(({ context, field, direction, showReset, o
             finalOptions = field.options || [];
           }
 
+          const onSelect = (selectedList:object, selectedItem:object) => {
+            let selection = [];
+            var parsedJson = JSON.parse(JSON.stringify(selectedList))
+            for(var i = 0; i < parsedJson.length; i++) {
+              selection.push(JSON.stringify(parsedJson[i].value));
+            };
+            changeCallback(field.name, selection.toString().replace(/"/g, ''))
+          };
+
+          const onRemove = (selectedList:object, removedItem:object) => {
+            let selection = [];
+            var parsedJson = JSON.parse(JSON.stringify(selectedList))
+            for(var i = 0; i < parsedJson.length; i++) {
+              selection.push(JSON.stringify(parsedJson[i].value));
+            };
+            changeCallback(field.name, selection.toString().replace(/"/g, ''))
+          };
+
           return (
-            <select {...inputProps()}>
```

Comment:
For non multi select, maybe we should use a simple selectbox?

Reply:
I did originally, but seemed redundant to me as I could use the same component for both where multiple lines were originally duplicated across both components and this reduced the lines of code needed and simplified the code.  Also, with this approach I felt it is better for the end-user to not have to think about other types and to just add a simple multi switch to the select type.